spec/abci/abci++_comet_expected_behavior.md

```diff
@@ -155,9 +155,9 @@ Let us now examine the grammar line by line, providing further details.
 >consensus-round     = proposer / non-proposer
 >```
 
-* For every round, if the local process is the proposer of the current round, CometBFT calls `PrepareProposal`, followed by `ProcessProposal`. 
+* For every round, if the local process is the proposer of the current round, CometBFT calls `PrepareProposal`, followed by `ProcessProposal`.
 These two always come together because they reflect the same proposal that the process
-also delivers to itself. 
+also delivers to itself.
```

Comment:
I think we need to adapt this as well

Reply:
Also, in the end, we agreed not to touch line 59 above, didn't we?
I think the final decision was to leave that line untouched (but not 100% sure, hence double-checking)

Reply:
I didn't change anything in this file because the grammar specifies the behavior in correct processes (that's why we agreed not to change line 59).
The main change hapened in the `example_scenarios` file, but I have now updated the text here. Let me know if I should revert.